lib/stitches/api_generator.rb

```diff
@@ -62,16 +63,17 @@ def bootstrap_api
       sleep 1 # allow clock to tick so we get different numbers
       migration_template "db/migrate/create_api_clients.rb", "db/migrate/create_api_clients.rb"
 
-      inject_into_file 'spec/spec_helper.rb', %q{
+      inject_into_file 'spec/rails_helper.rb', %q{
```

Comment:
So many of our internal apps were migrated from RSpec 2 and from a config that just has everything in `spec_helper` :(